CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# Release Date
+
+## Next version
+
+- MOEN-29601: AGP Version update from `8.0.2` to `8.2.2`
```

Comment:
For this repo alone,
Gradle version update from `8.1` to `8.2` will be better I think.
As the catalog has no reference to AGP.

Reply:
done